distributions/otelcol-k8s/.goreleaser.yaml

```diff
@@ -125,28 +125,6 @@ docker_manifests:
       - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-k8s:latest-arm64
       - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-k8s:latest-ppc64le
       - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-k8s:latest-s390x
-signs:
```

Comment:
to get this back, i think a few more of the `WithDefault*` methods need to be added to the distribution setup.
- `WithDefaultDockerSigns`
- `WithDefaultSBOMs`
- `WithDefaultSigns`